lib/storekit/client.rb

```diff
@@ -28,12 +28,13 @@ def verify!(base64_payload)
       resp = @http.request(req)
 
       decoded = JSON.parse resp.body
+      @last_response = ReceiptResponse.new(decoded)
```

Comment:
> This is broken because whether the response is a receipt or not depends on the value of the status field.

Exactly this was do there, I unbind returning response from the status field.

> This should behave as a pure function, so I disagree with storing the last response.

Why should, it is an action - method with the exclamation mark `!`, so it can modify the host object.